SkylinesPythonShared/SharedAPIStructures/Bezier.cs

```diff
@@ -51,16 +60,16 @@ public Vector flat_normal(float t)
 		public Bezier inverted => new Bezier(this.d, this.c, this.b, this.a);
 
 		[Doc("Point A (start)")]
-		public Vector a { get; private set; }
```

Comment:
I would implement custom [IXmlSerializable Interface](https://docs.microsoft.com/en-us/dotnet/api/system.xml.serialization.ixmlserializable?redirectedfrom=MSDN&view=net-6.0) instead of making the setters public. This way they would be accessible within the python script (at lest with the current IronPython engine) which probably is not good